fdbserver/GlobalTagThrottler.actor.cpp

```diff
@@ -641,7 +641,7 @@ class MockStorageServer {
 		Smoother smoother;
 
 	public:
-		Cost() : smoother(5.0) {}
+		Cost() : smoother(60.0) {}
```

Comment:
Is this meant to be the same as `GLOBAL_TAG_THROTTLING_FOLDING_TIME`? If so, could we use the knob here instead so it is easy to keep in sync with any future knob changes?